.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Set the default behaviour, in case poeple don't have core.autocrlf set
```

Comment:
This file is required because core.autocrfl is set to "auto" on Windows by default